sphinxcontrib_autodoc_filterparams/__init__.py

```diff
@@ -65,6 +65,8 @@ def _filter(param):
         return_annotation=inspect.Signature.empty
     )
 
+    _log.info(signature.format_args().replace('\\', '\\\\'), None)
```

Comment:
Why do we need to log the result if we also return the same value? If just for debugging purposes, then `_log.debug` seems more appropriate.